proto/decentraland/social_service/v2/social_service_v2.proto

```diff
@@ -425,6 +425,78 @@ message GetIncomingPrivateVoiceChatRequestResponse {
   }
 }
 
+// Community Voice Chat messages
+
+// Community voice chat credentials - specific type for community chats
+message CommunityVoiceChatCredentials {
+  string connection_url = 1;
+}
+
+// Starting a community voice chat
+message StartCommunityVoiceChatPayload {
+  string community_id = 1;
+}
+
+message StartCommunityVoiceChatResponse {
+  message Ok {
+    string voice_chat_id = 1;
+    CommunityVoiceChatCredentials credentials = 2;  // Moderator gets credentials immediately
+  }
+
+  oneof response {
+    Ok ok = 1;
+    InvalidRequest invalid_request = 2;
+    ForbiddenError forbidden_error = 3;
+    ConflictingError conflicting_error = 4;
+    InternalServerError internal_server_error = 5;
+  }
+}
+
+// Joining a community voice chat
+message JoinCommunityVoiceChatPayload {
+  string community_id = 1;
+}
+
+message JoinCommunityVoiceChatResponse {
+  message Ok {
+    string voice_chat_id = 1;
+    CommunityVoiceChatCredentials credentials = 2;
+  }
+
+  oneof response {
+    Ok ok = 1;
+    InvalidRequest invalid_request = 2;
+    ForbiddenError forbidden_error = 3;
+    NotFoundError not_found_error = 4;
+    ConflictingError conflicting_error = 5;
+    InternalServerError internal_server_error = 6;
+  }
+}
+
+// Getting active community voice chats for user's communities
+message GetCommunityVoiceChatsResponse {
```

Comment:
If we're able to get if a community has an active voice chat when requesting them via the HTTP endpoint, this should not be needed, what do you think about removing it?